book/Gremlin-Graph-Guide.adoc

```diff
@@ -14044,10 +14044,10 @@ that subgraph locally in memory while running some queries.
 Working with GraphML and GraphSON
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Apache TinkerPop supports loading and saving of entire graphs using GraphML and
+Apache TinkerPop supports the loading and saving of entire graphs using GraphML and

```

Comment:
Not sure this change is needed.

Reply:
I felt the "of" in this sentence either warranted the addition of `the` above, or removal; `Apache TinkerPop supports loading and saving entire graphs using GraphML...`. But I did make this change on intuition, so it can be disregarded.